packages/image_picker/image_picker_for_web/pubspec.yaml

```diff
@@ -21,7 +21,8 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  image_picker_platform_interface: ^2.2.0
+  image_picker_platform_interface: ^2.8.0
+  mime: ^1.0.4
```

Comment:
Capturing from Discord: this is Dart-team-owned, so just needs an entry in the appropriate section of `script/config/allowed_unpinned_deps.yaml`.